react/components/EXPERIMENTAL_Select/Option.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import OptionProps from 'react-select'
+
+import IconCheck from '../icon/Check'
+
+const Option = (props: OptionProps) => {
+  const {
+    children,
+    getStyles,
+    isMulti,
+    innerProps,
+    innerRef,
+    isSelected,
+  } = props
+  return (
+    <div style={getStyles('option', props)} ref={innerRef} {...innerProps}>
+      <div className="flex">
+        {isMulti ? null : (
+          <span className="flex self-center pr3" style={{ width: '1rem' }}>
```

Comment:
This inline style is really necessary?

Reply:
Did you try using `w1` from tachyons?

Reply:
Damn, I didn't know this one! Worked like a charm 👌